src/test/java/com/yahoo/validatar/AppTest.java

```diff
@@ -42,6 +42,8 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import static org.mockito.Mockito.*;
```

Comment:
This one doesn't need it. I forgot to remove it. I had a test in here that was using it.